spec/system/claims/support/schools/mentors/remove_mentor_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "Remove a mentor from a school", type: :system, service: :claims do
+  let!(:mentor1) { create(:mentor, first_name: "Bilbo", last_name: "Baggins") }
+  let!(:mentor2) { create(:mentor, first_name: "Bilbo", last_name: "Test") }
+  let!(:school) { create(:claims_school, mentors: [mentor1, mentor2]) }
+  let!(:colin) { create(:claims_support_user, :colin) }
+
+  scenario "View a school's mentors as a support user" do
+    user_exists_in_dfe_sign_in(user: colin)
+    given_i_sign_in
+    when_i_visit_the_support_school_mentors_page(school)
+    when_i_see_a_list_of_the_schools_mentors
+    when_i_select_a_mentor_to_remove
+    when_i_click_on_remove_mentor
+    when_i_confirm_removal
+    then_i_expect_to_see_the_mentor_has_been_removed
+  end
+
+  private
+
+  def given_i_sign_in
+    visit sign_in_path
+    click_on "Sign in using DfE Sign In"
+  end
+
+  def then_i_expect_to_see_the_mentor_has_been_removed
+    expect(page).not_to have_content("Bilbo Baggins1")
```

Comment:
Shouldn't this be?
```suggestion
    expect(page).not_to have_content("Bilbo Baggins")
```

Reply:
Ahh OK, no worries. Ignore my comment

Reply:
I think it might be best to split this into 2 lines. 1 line for the full name and 1 for the TRN.

Reply:
Or signify that they're on separate lines.